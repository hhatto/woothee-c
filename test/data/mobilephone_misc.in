Mozilla/5.0 (jig browser core; SH03B)

Mozilla/4.0 (jig browser9i 1.5.0; F10B; 2004)

Mozilla/4.0 (jig browser9)

emobile/1.0.0 (H11T; like Gecko; Wireless) NetFront/3.4

IAC/1.0 (H31IA;like Gecko;OpenBrowser) WWW Browser/ver1.0

Mozilla/5.0 (H11T; like Gecko; OpenBrowser) NetFront/3.4

Huawei/1.0/H12HW/B000 Browser/Obigo-Browser/Q04A

Mozilla/5.0 (SymbianOS/9.2; U; Series60/3.1 NokiaE66-1/500.21.009; Profile/MIDP-2.0 Configuration/CLDC-1.1 ) AppleWebKit/413 (KHTML, like Gecko) Safari/413

Mozilla/5.0 (en-us) AppleWebKit/534.14 (KHTML, like Gecko; Google Wireless Transcoder) Chrome/9.0.597 Safari/534.14

Mozilla/5.0 (compatible; livedoor-Mobile-Gateway/0.02; +http://p.m.livedoor.com/help.html)
