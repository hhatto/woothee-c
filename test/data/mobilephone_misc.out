target: Mozilla/5.0 (jig browser core; SH03B)
name: jig browser
version: SH03B
os: jig
category: mobilephone

target: Mozilla/4.0 (jig browser9i 1.5.0; F10B; 2004)
name: jig browser
version: F10B
os: jig
category: mobilephone

target: Mozilla/4.0 (jig browser9)
name: jig browser
os: jig
category: mobilephone

target: emobile/1.0.0 (H11T; like Gecko; Wireless) NetFront/3.4
name: emobile
os: emobile
category: mobilephone

target: IAC/1.0 (H31IA;like Gecko;OpenBrowser) WWW Browser/ver1.0
name: emobile
os: emobile
category: mobilephone

target: Mozilla/5.0 (H11T; like Gecko; OpenBrowser) NetFront/3.4
name: emobile
os: emobile
category: mobilephone

target: Huawei/1.0/H12HW/B000 Browser/Obigo-Browser/Q04A
name: emobile
os: emobile
category: mobilephone

target: Mozilla/5.0 (SymbianOS/9.2; U; Series60/3.1 NokiaE66-1/500.21.009; Profile/MIDP-2.0 Configuration/CLDC-1.1 ) AppleWebKit/413 (KHTML, like Gecko) Safari/413
name: Safari
os: SymbianOS
category: mobilephone

target: Mozilla/5.0 (en-us) AppleWebKit/534.14 (KHTML, like Gecko; Google Wireless Transcoder) Chrome/9.0.597 Safari/534.14
name: Mobile Transcoder
version: Google
os: Mobile Transcoder
category: mobilephone

target: Mozilla/5.0 (compatible; livedoor-Mobile-Gateway/0.02; +http://p.m.livedoor.com/help.html)
name: Mobile Transcoder
version: livedoor
os: Mobile Transcoder
category: mobilephone
